### DISCUSSION TITLE
FlyPadOS V2 Issues

### DISCUSSION THREAD

**Discussion by muskysim** [flybywiresim/aircraft#5310] [Help] [answered]
The first problem I ran into was I could not find where to  enter my Simbrief username to load flightplans. I found where to link Navigraph account with FlyPad and as I have a Navigraph FMS data subscription I have  Navigraph account and of course Navigraph have bought Simbrief so I thought this might be the way to do it.

When successfully completing the process of linking to my Navigraph account I found  I still could not load Simbrief flight plans and more concerning I had access to Navigraph Charts which I do not have a subscription for. 

So my 2 questions are:- 

1. Where do I enter my Simbrief username?
2. How come I got access to Navigraph charts when I do not have a subscription?

Regards 

P.S. Thankyou for making so many major improvements to this default aircraft

**Comment by oldmanbishop** [comment, marked as answer]
Simbrief username is now in the settings menu (icon bottom left of the pad.)

**Comment by ImenesFBW** [comment]
Which charts did you request, there are a few charts available as demo even when you do not have an active subscription?

**Comment by muskysim** [comment]
I am not sure what charts it was showing. Now though when I start the FlyPad it just sits there saying it is loading but it never does.

I have found another couple of issues though, in one of the tabs there is someting with the heading OFP when I click on it the Flypad goes black like it has switched off and I can't get it back again. Is this something daft I am doing?

**Reply by ImenesFBW** [reply]
The Navigraph integration has been disabled at this moment, while we investigate a few issues.
